ipn/ipnlocal/local.go

```diff
@@ -501,6 +523,7 @@ func (b *LocalBackend) populatePeerStatusLocked(sb *ipnstate.StatusBuilder) {
 			ShareeNode:     p.Hostinfo.ShareeNode(),
 			ExitNode:       p.StableID != "" && p.StableID == b.prefs.ExitNodeID,
 			ExitNodeOption: exitNodeOption,
+			ExitNodeIP:     b.exitNodeIPLocked(),
```

Comment:
Oh, sorry, I just realized this on at the wrong level of the status. You're putting this on PeerStatus which is in a loop for each peer (line 485).

We want just one of these in the status, not per-peer.

But we already have the ExitNode bool on PeerStatus and the TailscaleIPs field on the Peer, so all this info is already in the status output isn't it?

This change is taking the original issue's FR a bit too literally probably.

They don't want to do a loop, I guess, in which case putting it top-level as a sibling of the `Self` attribute is probably best.

Reply:
I've moved the status up to Self which makes sense from a local status view of whether an exit node is used.

On peer and on Self, ExitNode is whether it is an exit node, not if its using one and what its using which is the goal of ExitNodeIP. It makes sense that we'd also show if the peers are set to use an exit node and what node that is. Looking at that, I'd need to add that field to Node. Does that make sense to add ExitNodeIP field to Node?